examples/tv-app/android/java/ContentAppAttributeDelegate.cpp

```diff
@@ -0,0 +1,139 @@
+            return "";
+        }
+        const char * respStr = env->GetStringUTFChars(resp, 0);
+        ChipLogProgress(Zcl, "ContentAppAttributeDelegate::Read got response %s", respStr);
+        //        FormatResponseData(handlerContext, respStr);
+        return respStr;
+    }
+    return "";
+}
+
+CHIP_ERROR ContentAppAttributeDelegate::Read(const chip::app::ConcreteReadAttributePath & aPath,
+                                             chip::app::AttributeValueEncoder & aEncoder)
+{
+    if (aPath.mEndpointId >= FIXED_ENDPOINT_COUNT)
+    {
+        JNIEnv * env = JniReferences::GetInstance().GetEnvForCurrentThread();
+
+        ChipLogProgress(Zcl, "ContentAppAttributeDelegate::Read being called for endpoint %d cluster %d attribute %d",
+                        aPath.mEndpointId, aPath.mClusterId, aPath.mAttributeId);
+
+        jstring resp =
+            (jstring) env->CallObjectMethod(mContentAppEndpointManager, mReadAttributeMethod, static_cast<jint>(aPath.mEndpointId),
+                                            static_cast<jint>(aPath.mClusterId), static_cast<jint>(aPath.mAttributeId));
+        if (env->ExceptionCheck())
+        {
+            ChipLogError(Zcl, "Java exception in ContentAppAttributeDelegate::Read");
+            env->ExceptionDescribe();
+            env->ExceptionClear();
+            //            FormatResponseData(handlerContext, "{\"value\":{}}");
+            return CHIP_NO_ERROR;
+        }
+        const char * respStr = env->GetStringUTFChars(resp, 0);
+        ChipLogProgress(Zcl, "ContentAppAttributeDelegate::Read got response %s", respStr);
+        //        FormatResponseData(handlerContext, respStr);
+        return CHIP_NO_ERROR;
+    }
+    return CHIP_NO_ERROR;
+}
+
+/*
+void ContentAppCommandDelegate::FormatResponseData(CommandHandlerInterface::HandlerContext & handlerContext, const char * response)
+{
+    Json::Reader reader;
+    Json::Value value;
+    reader.parse(response, value);
+
+    switch (handlerContext.mRequestPath.mClusterId)
+    {
+    case app::Clusters::ContentLauncher::Id: {
+        LaunchResponseType launchResponse;
+        if (value["0"].empty())
+        {
+            launchResponse.status = chip::app::Clusters::ContentLauncher::ContentLaunchStatusEnum::kAuthFailed;
+        }
+        else
+        {
+            launchResponse.status = static_cast<chip::app::Clusters::ContentLauncher::ContentLaunchStatusEnum>(value["0"].asInt());
+            if (!value["1"].empty())
+            {
+                launchResponse.data = chip::MakeOptional(CharSpan::fromCharString(value["1"].asCString()));
+            }
+        }
+        handlerContext.mCommandHandler.AddResponseData(handlerContext.mRequestPath, launchResponse);
+        handlerContext.SetCommandHandled();
+        break;
+    }
+
+    // case app::Clusters::TargetNavigator::Id:
+    //     break;
+
+    // case app::Clusters::MediaPlayback::Id:
+    //     break;
+
```

Comment:
can be removed?

Reply:
Added as marker for which other responses have to be de-serialized.